docker-compose.yml

```diff
@@ -1,11 +1,11 @@
 web:
   build: .
-  command: mix phoenix.server
+  command: service nginx start && mix phoenix.server
```

Comment:
I also wonder about the right place for these. In a common case, the same set-up (Dockerfile) is used for `web` and `sidekiq` services in the docker-compose.yml. So, does it make sense then to have no command in the Dockerfile and leave it to docker-compose.yml to specify? Or, maybe Dockerfile should have a default command (e.g., starting the `web` service), with overrides in the docker-compose.yml for multiple interacting services.